cli/version/version_tools.go

```diff
@@ -73,28 +73,40 @@ type Version struct {
 	BuildName  string  // Custom build name.
 }
 
-// Parse parses a version string and return the version value it represents.
-func Parse(verStr string) (Version, error) {
-	version := Version{}
-	var err error
-
-	// Part 1 (optional) -> custom build name,
+func createVersionRegexp(is_strict bool) *regexp.Regexp {
```

Comment:
camelCase for variable names only.
```suggestion
func createVersionRegexp(isStrict bool) *regexp.Regexp {
```